simcon_project/users/forms.py

```diff
@@ -1,15 +1,46 @@
-from django.contrib.auth.forms import UserCreationForm, UserChangeForm
 from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
 from .models import CustomUser
+from users.models import Student, SubjectLabel
+import django_tables2 as tables
+
 
 
 class CustomUserCreationForm(UserCreationForm):
 
-    class Meta(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
         model = CustomUser
         fields = ('email',)
 
 
+class NewStudentCreationForm(forms.Form):
```

Comment:
You might have already tried this, but we might be able to use the CustomUser model form for this, rather than specifying all these fields. Not crucial though.

Reply:
I kinda named it oddly but that form is actually for updating an already created student with the correct info that it was missing before. not creating a student.